standard/mock/mock_lpdb.lua

```diff
@@ -0,0 +1,97 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Mock/Lpdb
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local lpdbData = Lua.import('Module:Mock/Lpdb/Data', {requireDevIfEnabled = true})
+
+local mockLpdb = {}
+
+local _lpdb = {
+	lpdb = mw.ext.LiquipediaDB.lpdb
+}
+
+function mockLpdb.setUp()
+	mw.ext.LiquipediaDB.lpdb = mockLpdb.query
+end
+
+function mockLpdb.tearDown()
+	mw.ext.LiquipediaDB.lpdb = _lpdb.lpdb
+end
+
+--- Not yet support in Mock is:
+---- conditions with `OR` or `_`
+---- query with `::`
+---- order
+---- groupby
+function mockLpdb.query(table, parameters)
+	local data = lpdbData[table]
+
+	if not data then
+		return error(mw.message.new('liquipediadb-error-invalid-datatype'))
+	end
+
+	data = Array.map(data, function(entry)
+		return (Json.parseIfString(entry))
```

Comment:
```suggestion
		return Json.parseIfString(entry)
```

Reply:
Actually needed, want only the first parameter that's returned from `parseIfString`, not both. can make it into a variable to make it more clear.